comp/core/flare/flare.go

```diff
@@ -27,6 +29,7 @@ import (
 	"github.com/DataDog/datadog-agent/comp/core/flare/types"
 	"github.com/DataDog/datadog-agent/comp/core/log"
 	"github.com/DataDog/datadog-agent/comp/core/secrets"
+	rcsetting "github.com/DataDog/datadog-agent/comp/core/settings"
```

Comment:
This aliased import looks misleading. These are not RC settings. Can the alias be renamed?